gnomad/utils/file_utils.py

```diff
@@ -100,23 +102,62 @@ def parallel_file_exists(fpaths: List[str], parallelism: int = 750) -> Dict[str,
     )
 
 
-def file_exists(fname: str) -> bool:
+def file_exists(fname: str, overwrite: Optional[bool] = None) -> bool:
     """
     Check whether a file exists.
 
```

Comment:
I feel that the `overwrite` parameter needs use case explanation, since overwrite is kind of out of the functions scope (just to check if file exists) and it causes some confusion. Maybe just me, so it's up to you :)
```suggestion
The function will optionally raise an error if `overwrite` is set to False and file exists, when the the function is called to check if the new created file path exist.
```

Reply:
Let me know if the changes I made works. They will work for my need